vignettes/how-to-update-released-site.Rmd

```diff
@@ -94,15 +102,15 @@ usethis:::git_push_first()
 
 Then trigger the pkgdown workflow:
 
-1.  Go to your package's GHA page, e.g. with `usethis::browse_github_actions())`.
+1.  Go to your package's GHA page, e.g. with `usethis::browse_github_actions()`.
 2.  Select the pkgdown workflow.
 3.  Click *Run workflow* and select the branch you just pushed.
 
 If there's no dropdown menu for this, that means your pkgdown workflow config is not current.
 
 ## Context
 
-Before we talk about **how** to update a released site, we first establish **why** you might need to do this.
+Now that we've sketched **how** to update a released site, we discuss **why** you might need to do this.
```

Comment:
This article got rearranged at some point and this sentence no longer made sense.